packages/flutter/lib/src/foundation/core_io.dart

```diff
@@ -1,49 +1,87 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
+// Copyright 2015 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: public_member_api_docs
```

Comment:
I'm not sure what the best solution is here. Ideally we could enforce something in the analyzer to skip docs

Reply:
Essentially, I would expect that a top level member in a conditionally imported library behaves similarly to an overriden method.